### ISSUE TITLE
Add Help Button To Merge Editor

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#159159]
FYI @daviddosset @isidorn

**Comment by isidorn** [comment]
Please not for August :)
Let's discuss and see what we can do for September.

**Comment by daviddossett** [comment]
Something lightweight like this? Or did you have something else in mind?

My sense is that most people won't see this. Do we have other patterns used for drawing attention to help content elsewhere? cc @digitarald since I think we've talked about this kind of thing in the past.

<img width="1256" alt="CleanShot 2022-09-07 at 15 07 36@2x" src="https://user-images.githubusercontent.com/25163139/188992278-3b9f1ee7-ca8d-48d2-8d60-97cc17dc5527.png">

**Comment by digitarald** [comment]
I love how versatile the style is for future lean-more occasions.

To solve the attention problem (it does appear in a very busy UI), maybe it could animate on first open to draw attention? Prior art would be the language indicator jiggle, but maybe we can also do color or scale animations.

Alternatively, a tooltip (using the existing built-in style) could be attached to the icon on first open that disappears after a short moment.

**Comment by isidorn** [comment]
After discussing with @hediet and @jrieken we are leaning towards an editor banner similar to the one we use for the ambiguous characters.
This banner needs to have:
* Helpful text
* Help link - would link to video or docs @burkeholland
* Disable link - would open settings editor with git.mergeEditor setting
* X button to close the banner

![Screenshot 2022-09-08 at 15 46 15](https://user-images.githubusercontent.com/1926584/189138646-6d0053ae-3f4a-4543-bb28-1c7c4bb4caff.png)

**Comment by daviddossett** [comment]
This could work. It will have the side effect of making the overall UI feel even more cluttered at first.

**Comment by isidorn** [comment]
@daviddossett yeah it is a tradeoff. On the plus side it is easy to hide. Any other UI we add is not so easily hide-able.
